src/modules/integrator-guide/pages/compose-configuration/duplication-detection.adoc

```diff
@@ -5,10 +5,122 @@ include::ROOT:partial$variables.adoc[]
 {PRODUCT_NAME} through the duplicate detection feature helps maintain clean and accurate data by preventing record duplication.
 This is achieved by using rules to apply a criterion that determines how closely a field on a new or edited record matches the same field on an existing record.
 
-The user interface displays a warning or an error message to alert the user of an existing record with the exact field values within a module thereby preventing duplicates.
+The user interface displays a warning or an error message to alert the user of an existing record with the exact field values within a module, thereby preventing duplicates.
 
 .Benefits of corteza duplicate detection:
 * Maintain data quality,
 * improves accuracy and efficiency in data,
 * enhances effective description of records,
 * optimizes data storage.
+
+== Validation Rules
+
+The duplicate detection feature is enabled by using the value validation rules on a module.
+Validation rules verify that the designated fields in a record do not contain the same values, therefore ensuring that no duplicate values are saved when creating or editing a record.
+
+Validation can be configured using two rules:
+
+* Strict duplicate value validation
+
+This rule ensures that a record will not be saved when a duplicate value is detected in the selected fields in any existing record of this module. 
+
+When the Strict duplicate value validation rule is applied, an error message pop-up will be displayed on the bottom right side of the screen as an indication that a record with the same values exists.
+
+* Soft duplicate value validation
+
+This rule allows a record to be saved but issues a warning message pop-up when a duplicate value is detected, indicating that there is an existing record with the exact field values within the module.
+
+When Soft duplicate value validation is applied, the warning message is displayed on the bottom right side of the screen.
+
+=== Enabling value validation rules:
+
+[NOTE]
+====
+Validation rules can be enabled when creating new modules or on exiting modules.
+====
+
+Navigate to your Low Code namespace. Click on an existing module or create a new module.
+
+On the given module, navigate to the *[validation]* tab.
+
+[annotation,role="data-zoomable"]
+----
+{
+  "image": {
+    "rel": "duplication-detection/duplicate-detection-validation.png",
+    "alias": "duplicate-detection-validation",
+    "w": 1920,
+    "h": 1080
+  },
+  "view": {},
+  "annotations": [{
```

Comment:
Your annotations are a bit off-aligned. Let me explain...

This is one of your annotations -- notice how it's a bit off aligned compared to where the tab is (what you're pointing out)

![image](https://user-images.githubusercontent.com/12545711/196209398-29a50cab-de4f-41fa-af3d-d7b5db2a3578.png)

What you need to do is open the image in new tab, open element inspector, and find the `<rect>` SVG element with `visibility=hidden`.
Remove the `hidden` part and you'll get a red box.

![image](https://user-images.githubusercontent.com/12545711/196209487-820684f3-4231-406f-97ae-fb54504d3288.png)

Align the red box to the element you're trying to annotate and use those coordinates and dimensions in your annotations.
So in this case, the aligned box would look like this

![image](https://user-images.githubusercontent.com/12545711/196209717-25dd70e7-5538-4fbf-a34d-bc5631565b22.png)

and then, when rendered, like this

![image](https://user-images.githubusercontent.com/12545711/196210211-a3a72b2a-5264-4ae7-9b5a-f4f63b98bbea.png)

This helps assure consistency and makes it look good in general.
There are docs regarding this, you should have them already.

Reply:
Please do the same for all annotations and make sure they are aligned to what they are calling out

Reply:
okay